packages/sync-service/lib/electric/shapes/consumer_registry.ex

```
@@ -120,6 +120,8 @@ defmodule Electric.Shapes.ConsumerRegistry do
 
   defp consumer_pid(handle, table) do
     :ets.lookup_element(table, handle, 2, nil)
+  rescue
+    ArgumentError -> nil
```

if this returns nil, it'll try to start a consumer and fail, so you'll still get an exception. I guess your argument is that it's an easier to interpret error message? I would argue that it's better to raise here with a nice readable message, that way you get your readable message but without swallowing the error which could cover up bugs